core/templates/pages/contributor-dashboard-page/question-badge/question-badge.component.html

```diff
@@ -0,0 +1,41 @@
+<div *ngIf="isUnlocked === 'true'" class="badge-container">
+    <img class="translation-badge" src="/assets/images/contributor_dashboard/question-{{type}}-badge.component.svg">
+    <div class="badge-value">
+        {{value}}
+    </div>
+    <div class="badge-caption">
+        {{contributionTypeText}}
+    </div>
+</div>
+<div *ngIf="isUnlocked === 'false'"  
```

Comment:
```suggestion
<div *ngIf="!isUnlocked"  
```

Should work since `isUnlocked` is a boolean. Here and elsewhere.

Reply:
Done